### DISCUSSION TITLE
local host not permitted

### DISCUSSION THREAD

**Discussion by Vulpesx** [cc-tweaked/CC-Tweaked#626] [Q&A] [answered]
## Useful information:
 - Minecraft version: 1.16.4
 - CC: Tweaked version: 1.94.0
 
I have a startup file to connect to a websocket server running in Java
*startup.lua*  ` ws, err = https.websocket("ws://localhost:43509")`
*error* `Domain not permitted`
the sever doesn't even receive the websocket

this is probably a very obvious fix but hey it's my first experience with websockets

**Comment by GuilAFT** [comment, marked as answer]
It's not a bug, local IP addresses are not permitted due to security issues. You can disable this by removing 127.0.0.1
from the computercraft-server file located in the serverconfig folder of your save.

**Comment by Vulpesx** [comment]
thanks :)

**Comment by Vulpesx** [comment]
that ip isn't in the config

**Comment by xAnavrins** [comment]
That IP is in there by default, you only need to change `action = "deny"` to `action = "allow"`
It should look like this.
```
[[http.rules]]
	host = "127.0.0.0/8"
	action = "allow"
```

**Comment by Vulpesx** [comment]
yes I tried that but it was just changed back when I re launched the game

**Reply by HenryDu8133** [reply]
You need to modify it when you close the game

**Comment by Vulpesx** [comment]
Oh, i tried it again and it worked, why it didn't work the first time idk but yay :)

**Reply by Lupus590** [reply]
you modified the file while the game was open and when it restarted it saved the old version of the file over your edits
